configuration/ingress/alb/README.md

```diff
@@ -0,0 +1,41 @@
+# ALB (Application Load Balancer) Ingress
+
+In addition to the main [setup](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html) - when using AWS Application Load Balancer (ALB) for ingress, certain annotations are required to ensure proper routing and load balancing.
+
+## Key Annotations
+
+The following additional annotations are required, otherwise multiple ALBs will be processing rules for the same host name.  The group order ensures that the most specific rules are processed first.
```

Comment:
Where do these annotations go?  Are there Helm chart settings to inject them?  Or do you need to `kubectl edit` or otherwise post-process specific resources?